frontend/doc-site/09_Frequently-Asked-Questions.mdx

```diff
@@ -2,4 +2,59 @@
 
 We’ve compiled a list of the questions we get most frequently. Check them out below:
 
-**MAX**
+
+## Cellxgene Data Portal
+
+### How can I find a dataset of interest?
+
+Refer to this [tutorial (TODO)](www.google.com) to find out how the dataportal is organized and how you can search by dataset metadata.
+
+### How can I request a particular dataset to be on the portal?
+
+You can request a particular dataset by [filing a github issue](https://github.com/chanzuckerberg/single-cell-curation/issues/new) on the cellxgene single-cell-curation repo.
+
+### How can I contribute a dataset to the cellxgene data portal?
+
+We are continuing to expand our data corpus and would love to chat further. Reach out to our curation team at [cellxgene@chanzuckerberg.com ](cellxgene@chanzuckerberg.com )
+
+## Cellxgene hosted Explorer
+
+### Is there a way to shift the color scales for continuous variables?
+
+You can use the clipping feature to remove outliers (based on percentiles) and shift the color scale (see [this tutorial](https://docs.google.com/presentation/d/13c0Nj_kR32j0hNL0uZG_4cl646N_TH3ADaQzZFqDPBI/edit#slide=id.gee6c9f6165_0_18) for a demonstration of the feature on slide 8)
```

Comment:
Probably won't expose access, but this is an edit link!

Reply:
yea - this the link that google gives me (even when I ask for a link to view only) - do you still think it is an issue?

Reply:
looks good to me!